File: 250827/짝수만 거꾸로 출력하기/print-even-numbers-upside-down.py
N = int(input())
arr = map(int, input().split())

result = []
for i in arr:
    if (i % 2 == 0):
        result.append(i)

for j in range(len(result) - 1, -1, -1):
    print(result[j], end=" ")

File: 250827/짝수만 거꾸로 출력하기/print-even-numbers-upside-down.js
const fs = require("fs");
let arr = fs.readFileSync(0).toString().trim().split(/\s+/).map(Number);

let result = "";
for (let i = arr.length -1; i > 0; i--) {
    if (arr[i] % 2 == 0) {
        result += arr[i] + " ";
    }
}
console.log(result);